samples/system-test/v3beta1/translate_delete_glossary_beta.test.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {TranslationServiceClient} = require('@google-cloud/translate').v3beta1;
+const execa = require('execa');
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+
+const REGION_TAG = 'translate_delete_glossary_beta';
+
+describe(REGION_TAG, () => {
+  const translationClient = new TranslationServiceClient();
+  const location = 'us-central1';
+  const glossaryId = 'glossary';
+
+  before(async function() {
+    // Add a glossary to be deleted
+    // const translationClient = new TranslationServiceClient();
+    const projectId = await translationClient.getProjectId();
+    const glossary = {
+      languageCodesSet: {
+        languageCodes: ['en', 'es'],
+      },
+      inputConfig: {
+        gcsSource: {
+          inputUri: 'gs://cloud-samples-data/translation/glossary.csv',
+        },
+      },
+      name: translationClient.glossaryPath(projectId, location, glossaryId),
+    };
+
+    // Construct request
+    const request = {
+      parent: translationClient.locationPath(projectId, location),
+      glossary: glossary,
+    };
+
+    // Create glossary using a long-running operation.
+    // You can wait for now, or get results later.
+    const [operation] = await translationClient.createGlossary(request);
+
+    // Wait for operation to complete.
+    await operation.promise();
+  });
+  
+  it('should delete a glossary', async () => {
+    const projectId = await translationClient.getProjectId();
+
+    const output = await exec(
```

Comment:
I think for simple assertions like this, you might be able to get away with `execSync` moving away from `execa`, it works like so:

```
const {execSync} = require("child_process")
execSyng(process.execPath, [`v3beta1/${REGION_TAG}.js`, `${projectId` ...)
```

Reply:
@bcoe if we're gonna make a change like that, it should be discussed in `google-cloud-node`, and we should pass down guidance that way.  For now lets stick with execa for consistency pls :)

Reply:
+1 to both of ya

At some point in Q2, after these are all replaced with robot generated ones, I'll work with y'all and you'll be able to tweak the templates to your heart's content as long as y'all want ;)